File: LeetCode/896. Monotonic Array/monotonicArray.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        increasing: bool = True
        decreasing: bool = True
        length: int = len(nums) - 1
        for i in range(length):
            if nums[i] < nums[i+1]:
                decreasing = False
        for i in range(length):
            if nums[i] > nums[i+1]:
                increasing = False
        return increasing or decreasing

File: LeetCode/896. Monotonic Array/monotonicArray.cs
public class Solution {
    public bool IsMonotonic(int[] nums) {
        bool increasing = true;
        bool decreasing = true; 
        int length = nums.Length;
        for (int i = 0; i < length - 1; i++) {
            if (nums[i] < nums[i + 1])
                decreasing = false;
        }
        for (int i = 0; i < length - 1; i++) {
            if (nums[i] > nums[i + 1])
                increasing = false;
        }
        return increasing||decreasing;
    }
}
